modules/protect.php

```diff
@@ -423,11 +423,16 @@ function ip_is_whitelisted( $ip ) {
 	 * @return bool Either returns true, fires $this->kill_login, or includes a math fallback and returns false
 	 */
 	function check_login_ability( $preauth = false ) {
+		$ip = jetpack_protect_get_ip();
+		
+		if ( apply_filters( 'jpp_allow_login', false, $ip ) ) {
```

Comment:
Filter name is 👍 by me, if you can get a docblock in, I'm happy to merge in for 4.4